src/pages/why.tsx

```diff
@@ -0,0 +1,412 @@
+                        value: 'PostHog is literally designed to be a no-brainer',
+                        depth: 0,
+                    },
+                    {
+                        url: 'popular-roadmap-items',
+                        value: 'Popular roadmap items',
+                        depth: 0,
+                    },
+                    {
+                        url: 'posthog-has-the-lowest-pricing-for-every-product',
+                        value: 'PostHog has the lowest pricing for every product',
+                        depth: 0,
+                    },
+                    {
+                        url: 'posthog-just-does-the-right-thing',
+                        value: 'PostHog just does the right thing',
+                        depth: 0,
+                    },
+                    {
+                        url: 'support-from-actually-technical-people',
+                        value: 'Support from actually technical people',
+                        depth: 0,
+                    },
+                    {
+                        url: 'posthog-is-more-popular-than-every-other-product',
+                        value: 'PostHog is more popular than every other product',
+                        depth: 0,
+                    },
+                    {
+                        url: 'who-doesnt-posthog-work-well-for',
+                        value: "Who doesn't PostHog work well for?",
+                        depth: 0,
+                    },
+                    {
+                        url: 'every-product-has-a-huge-free-tier',
+                        value: 'Every product has a huge free tier',
+                        depth: 0,
+                    },
+                ]}
+            >
+                <section className="mx-auto">
+                    <div className="flex flex-col md:items-center md:justify-end md:flex-row-reverse gap-8 md:gap-2">
+                        <div className="md:flex-1">
+                            <div className="@container article-content">
+                                <div className="opacity-70 text-[15px] font-medium">Why PostHog?</div>
+                                <h1 id="posthog-is-literally-designed-to-be-a-no-brainer" className="text-2xl">
+                                    PostHog is literally designed to be a no-brainer
+                                </h1>
+                                <p>
+                                    <strong>
+                                        PostHog is an entire suite of products you can use to make your software
+                                        successful – <em>far more than anyone else.</em>
+                                    </strong>
+                                </p>
+                                <p>
+                                    All that context in one spot so you can focus on your users and not your data
+                                    engineering to fix endless system integrations.
```

Comment:
```suggestion
                                    We put all the context you need in one place, so you can focus on your users instead of fixing 
                                    endless data integrations.
```